tests/integration/pybaseball/test_playerid_lookup.py

```diff
@@ -30,4 +30,33 @@ def test_playerid_lookup_phonetic_bogaerts() -> None:
     # Xander Bogaerts
     bogaerts_df = playerid_lookup("bogarts", "zander", fuzzy=True)
     assert bogaerts_df["name_last"][0] == "bogaerts"
-    assert bogaerts_df["name_first"][0] == "xander"
\ No newline at end of file
+    assert bogaerts_df["name_first"][0] == "xander"
+    
+def test_playerid_lookup_three_word_name() -> None:
+    """Test names with three words in them"""
+    # Hyun Jin Ryu
+    ryu_df = playerid_lookup("jin ryu", "hyun", fuzzy=True)
+    assert ryu_df["name_last"][0] == "ryu"
+    assert ryu_df["name_first"][0] == "hyun jin"
+
+def test_playerid_lookup_abbreviated_name() -> None:
+    """Test names with abbreviations in them"""
+    # JD Martinez
+    martinez_df = playerid_lookup("martinez", "jd", fuzzy=True)
+    assert martinez_df["name_last"][0] == "martinez"
+    assert martinez_df["name_first"][0] == "j. d."
+    
+def test_playerid_lookup_name_with_jr() -> None:
+    """Test names with abbreviations in them"""
+    # Ronald Acuna Jr
+    acuna_df = playerid_lookup("acuna jr.", "ronald", fuzzy=True)
+    assert acuna_df["name_last"][0] == "acuna"
+    assert acuna_df["name_first"][0] == "ronald"
+    
+def test_playerid_lookup_hyphenated_name() -> None:
+    """Test names with abbreviations in them"""
+    # Isiah Kiner-Falefa
+    acuna_df = playerid_lookup("Kiner Falefa", "isiah", fuzzy=True)
```

Comment:
`acuna_df` doesn't look like an accurate name for this test, could you change it?

Reply:
My bad. Just pushed the changes.